apps/user-office-frontend/src/components/user/ExternalAuth.tsx

```diff
@@ -146,7 +146,8 @@ function ExternalAuth() {
     setView(<LoadingMessage />);
 
     const errorDescription = urlQueryParams.error_description;
-    const authorizationCode = urlQueryParams.sessionid ?? urlQueryParams.code;
+    const authorizationCode =
+      urlQueryParams.sessionid ?? urlQueryParams.code ?? urlQueryParams.token;
```

Comment:
Do you need `sessionid` ? We only use the `code`

Reply:
I don't think we do still use it but I'm not 100% sure. Removing this line and the login still works:

https://github.com/UserOfficeProject/user-office-core/blob/develop/apps/user-office-frontend/src/components/App.tsx#L62

If we're in doubt then I think we could leave this condition as is for now and it shouldn't affect anything, right? As it'll just default to the others if null.